data/systems.js

```diff
@@ -0,0 +1,5 @@
+module.exports = [
+  { 'console': 'PlayStation3' },
```

Comment:
just FYI, you may have been having issues with this needing to be in quotes because `console` is a reserved word in Node / Javascript